ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -838,3 +838,69 @@ def test_torch_unique(
         return_counts=return_counts,
         dim=axis,
     )
+
+
+@st.composite
+def _get_axis_and_p(draw):
+    force_tuple_axis = True
+
+    p = draw(st.sampled_from(["fro", "nuc", 1, -1, 2, -2]))
+    if p == "fro" or p == "nuc":
+        force_tuple_axis = True
+        min_axes_size = 2
+
+    else:
+        force_tuple_axis = False
+        min_axes_size = 1
+
+    dtype_x_axis = draw(
+        helpers.dtype_values_axis(
+            num_arrays=1,
+            available_dtypes=helpers.get_dtypes("float"),
+            min_num_dims=2,
+            max_num_dims=3,
+            max_dim_size=2,
+            valid_axis=True,
+            min_axes_size=min_axes_size,
+            large_abs_safety_factor=2,
+            safety_factor_scale="log",
+            force_tuple_axis=force_tuple_axis,
+        )
+    )
+
+    return (p, dtype_x_axis)
+
+
+# norm
+@handle_frontend_test(
+    fn_tree="torch.norm",
+    p_dtype_x_axis=_get_axis_and_p(),
+    keepdim=st.booleans(),
+    dtype=helpers.get_dtypes("float", full=False),
+)
+def test_torch_norm(
+    *,
+    p_dtype_x_axis,
+    keepdim,
+    dtype,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    p, values = p_dtype_x_axis
+    input_dtype, x, axis = values
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-01,

```

Comment:
Looks good to me! Only thing is the `rtol` 10% is very high. Isn't the test passing without this?

Reply:
@AnnaTz
 yes because when I change it it say the ground truth between frame works not equal but when I test it it give the same result I don't know why

Reply:
Then I guess there is some inconsistency between torch and our ivy functions. I don't think this can be fixed in this PR. I'd say you can merge this now.